testing/kuttl/e2e/exporter/10-assert.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: exporter-tls
+status:
+  instances:
+    - name: instance1
+      readyReplicas: 1
+      replicas: 1
+      updatedReplicas: 1
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: exporter-tls
+    postgres-operator.crunchydata.com/pgbackrest-backup: replica-create
+status:
+  succeeded: 1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: exporter-tls-primary
```

Comment:
🤔 I don't see this Service being used (port, DNS, etc) in later steps. What does the assertion mean here?

Reply:
I thought these were good general checks to ensure that the cluster had started successfully. I don't think they are specifically necessary so could be removed